src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -746,18 +770,24 @@ public String getQuestionResultStatisticsCsv(
         DecimalFormat dfAverage = new DecimalFormat("0.00");
 
         int[][] responseFrequency = calculateResponseFrequency(responses, this);
-        float[][] rubricStats = calculateRubricStats(responses, question);
-        
+        float[][] rubricStats = calculateRubricStats(responses, this);
```

Comment:
Hey perhaps it would also be good to add `responseFrequency` as a param to `calculateRubricStats`, to avoid having to calculate it again. You could do in in a separate PR if you'd like.